src/encoded/static/components/genome_browser.js

```diff
@@ -554,10 +596,11 @@ class GenomeBrowser extends React.Component {
 
     setGenomeAndTracks() {
         const genome = mapGenome(this.props.assembly);
+        const annotation = mapGenome(this.props.annotation);
```

Comment:
Does `mapGenome` do anything with the annotation? Is this in case we need it in the future? Also, the first line of `mapGenome` splits the parameter, which I don’t think you need to do anymore.

Reply:
You are correct. No need. I didn't think carefully and just copied the behavior from assembly.